www/source/blog/2018-03-21-0550-Release.html.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Habitat 0.55.0 Released
+date: 2018-03-21
+author: Elliott Davis
+tags: release notes
+category: product
+classes: body-article
+---
+
+# Habitat 0.55.0 Release Notes
+
+We are happy to announce the release of Habitat 0.55.0. We have a number of new features as well as bug fixes, so please read on for all the details. If you just want the binaries, head on over to [Install Habitat](https://www.habitat.sh/docs/using-habitat/#install-habitat). Thanks again for using Habitat!
+
+## Highlights:
+
+* [Feature] Ability to create and upload secrets via the hab cli
+* [Feature] Improve startup time of Windows container studios
+* [Feature] Windows container Studio runs its background supervisor as a service
+* [Bugfix] Windows Supervisor honors a Plan's pkg_svc_user
+* [Bugfix] Windows container Studio correctly streams Supervisor output from Get-SupervisorLog
+* [Bugfix] Updated default.toml files are seen on package update (fixes https://github.com/habitat-sh/habitat/issues/4576)
+* [Bugfix] Launcher now reaps orphaned zombie processes (fixes https://github.com/habitat-sh/habitat/issues/4412)
+
+### Habitat Origin Secrets
+
+In a previous release we enabled users to pass in secrets to the studio. With this release, you can now create secrets for your Habitat origin in Builder. These secrets are encrypted locally before they are uploaded to Builder. *Only* Builder can decrypt these secrets. Once a secret is attached to your origin it will be passed into your Builder jobs as part of the build. This is really useful for things like ssh keys, AWS credentials etc.
+
+#### Usage
+
+`hab origin secret upload SECRET_KEY secret_value`
+`hab origin secret list`
+`hab origin secret delete SECRET_KEY`
+
+### Habitat Launcher: new version released, please update
```

Comment:
Super minor, but should probably be title-capped: `New Version Released, Please Update`, or at least with a capped `N`.